node/runtime/src/lib.rs

```diff
@@ -393,6 +393,7 @@ impl sudo::Trait for Runtime {
 }
 
 impl im_online::Trait for Runtime {
+	type AuthorityId = ImOnlineId;
```

Comment:
Just to be clear, we are importing sth from the im-online module and feeding it again to the module's trait, right? is this like a sort of a default?

Reply:
No, here we are fixing the crypto to be sr25519. If you check `ImOnlineId` is imported from the `sr25519` package, we could as well import and use `ed25519` instead now.